projects/dotcms-field-elements/src/components/dot-form/models/dot-type-field-column.model.ts

```diff
@@ -0,0 +1,6 @@
+import { DotCMSContentTypeField } from '.';
+
+export interface DotCMSContentTypeColumn {
+    columnDivider?: DotCMSContentTypeField;
```

Comment:
Isn't this named `divider` in the backend?

Reply:
![image](https://user-images.githubusercontent.com/37185433/59869910-a7e5cc80-9351-11e9-9f2f-f31bbf9cabab.png)
that's what was provided in the mock response